tests/treeTest.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from coalahtml.tree import Node, Tree
+from coalib.coala_main import run_coala
+from coalib.output.printers.ListLogPrinter import ListLogPrinter
+from coalahtml.helper import parse_file_dict
+
+
+class TreeTest(unittest.TestCase):
+
+    def setUp(self):
+        log_printer = ListLogPrinter()
+        file_dict = run_coala(log_printer=log_printer,
+                              autoapply=False)[2]
```

Comment:
Then use real one directory structure, but add it by hand. Tests should not be sophisticated.